pam/pam-module-transaction-dummy.go

```diff
@@ -0,0 +1,295 @@
+package main
```

Comment:
This file is mainly for defining the `PamModuleTransactionDummy` type, so I would switch this from `package main` to `package pam` instead, as `package main` is related to an executable code instead (i.e. having a Main() func and so on).